pkg/eval/platform/platform.go

```diff
@@ -0,0 +1,29 @@
+// Package `os` exposes variables and functions that deal with the specific
+// platform being run on; such as the OS name and CPU architecture.
+package platform
+
+import (
+	"runtime"
+
+	"github.com/elves/elvish/pkg/eval"
+	"github.com/elves/elvish/pkg/eval/vars"
+)
+
+//elvdoc:var arch
+//
+// The architecture of the platform; e.g., amd64, arm, ppc.
```

Comment:
Remove the comma after `e.g.`.

Reply:
Blame my elementary school teachers :-) They taught me that both e.g. and i.e. should be followed by a comma. Googling "should e.g. have a comma" turns up a lot of conflicting advice with most sources saying it's okay either way. But, happy to oblige as this is a bike shedding issue :-)

Reply:
Interesting, I never knew that.